torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1071,7 +1090,9 @@ def _calculate_outlier_info(self, percentile_ratios: torch.Tensor, counted_batch
         num_batches_list: List = counted_batches.tolist()
 
         # calculate whether channels were statistically significant
-        significant_size = [True if batch_size >= 30 else False for batch_size in num_batches_list]
+        significant_size = [
+            True if batch_size / total_batches >= self.statistical_threshold else False for batch_size in num_batches_list
```

Comment:
Can simplify to `[batch_size / total_batches >= ... for batch_size in num_batches_list]`